packages/modular-scripts/src/build.ts

```diff
@@ -197,15 +198,32 @@ async function makeBundle(
       `${packagesRoot}/${packagePath} is marked private, bailing...`,
     );
   }
-  if (!packageJson.main) {
+
+  let compilingBin = false;
+  let main: string;
+  if (!packageJson.main || !packageJson.bin) {
```

Comment:
This is failing the tests 
```suggestion
  if (!packageJson.main && !packageJson.bin) {
```